test/unit/v_all/test_replication.py

```diff
@@ -39,7 +39,11 @@ def setUp(self):
         self.application_key_id, self.master_key = self.raw_api.create_account()
 
     def _authorize_account(self):
-        self.api.authorize_account('production', self.application_key_id, self.master_key)
+        self.api.authorize_account(
+            realm='production',
```

Comment:
Nitpick: all other changes to `kwargs` resulted in placing `realm` being placed as last in the invocation.